front/src/common/api/osrdEditoastApi.ts

```
@@ -186,6 +188,40 @@ const osrdEditoastApi = generatedEditoastApi
         // TODO: fix getPacedTrainByIdSimulation tags (it should be paced_train)
         providesTags: ['train_schedule'],
       }),
+      getTimetableItemsOperationalPoints: builder.query<
+        OperationalPoint[][],
+        { infraId: number; opRefs: OperationalPointReference[] }
+      >({
+        queryFn: async ({ infraId, opRefs }, { dispatch }) => {
+          const batchSize = 200;
+          const result: OperationalPoint[][] = [];
+
+          // Split opRefs into batches of 200
+          for (let i = 0; i < opRefs.length; i += batchSize) {
+            const batch = opRefs.slice(i, i + batchSize);
+
+            const promise = dispatch(
+              osrdEditoastApi.endpoints.postInfraByInfraIdMatchOperationalPoints.initiate(
+                {
+                  infraId,
+                  body: {
+                    operational_point_references: batch,
+                  },
+                },
+                { forceRefetch: true, subscribe: false }
+              )
+            );
+
+            const { data } = await promise;
```

This ignores errors. `unwrap()` can be used to throw on error instead of silently dropping some results.

---

To be sure to understand, you would like to extract also `error` from `promise` and throw if we have one ?
It seems like we can access `error` without `.unwrap()`

---

There is no need to extract the error manually: `.unwrap()` throws if there is an error.

---

Oh didn't know about that ! Done.